### DISCUSSION TITLE
get html source

### DISCUSSION THREAD

**Discussion by Antonio-Prado** [browsh-org/browsh#471] [General]
hi,

is there a way to use browsh to get the html source (after js) of a page and to put it in a file?

thank you

**Comment by tombh** [comment]
I love lowercase typing, it expresses a form of informality. But I don't think requesting help on an open source project is the best place to use it. I'd personally prefer if you took the time to use proper capitalisation 🙇

Try: `browsh --http-server-mode`
Docs: https://www.brow.sh/docs/in-browser-usage/

**Reply by Antonio-Prado** [reply]
thank you tom for your kind reply, for your time, and for your effort in this project
